src/duty/attestation/attestation.service.ts

```diff
@@ -79,14 +84,17 @@ export class AttestationService {
       this.getCanonSlotRoot(attestation.target_epoch * this.slotsInEpoch),
       this.getCanonSlotRoot(attestation.source_epoch * this.slotsInEpoch),
     ]);
-    const att_valid_head = attestation.head == canonHead;
-    const att_valid_target = attestation.target_root == canonTarget;
-    const att_valid_source = attestation.source_root == canonSource;
-    const att_inc_delay = Number(attestation.included_in_block - attestation.slot);
-    const flags = getFlags(att_inc_delay, att_valid_source, att_valid_target, att_valid_head);
+    const attValidHead = attestation.head == canonHead;
+    const attValidTarget = attestation.target_root == canonTarget;
+    const attValidSource = attestation.source_root == canonSource;
+    const attIncDelay = Number(attestation.included_in_block - attestation.slot);
+    const isDencunFork = epoch < this.dencunEpoch;
```

Comment:
```suggestion
    const isDencunFork = epoch > this.dencunEpoch;
```

Reply:
We discussed this internally and decided that checking `epoch >= this.dencunEpoch` should be enough here.
This `Math.floor(attestation.included_in_block / this.slotsInEpoch) === epoch` must always be true in the `processAttestation()` method, because if not, then it means that the whole method works incorrectly.